css/cssom-1/cssom-ruleType.html

```diff
@@ -0,0 +1,79 @@
+<!doctype html>
+<head>
+  <title>CSS OM: CSS Rule Types</title>
+  <link rel="author" title="Divya Manian" href="mailto:manian@adobe.com">
+  <link rel="author" title="Melanie Richards" href="mailto:Melanie.Richards@microsoft.com">
+  <link rel="help" href="https://www.w3.org/TR/cssom-1/#the-cssrule-interface">
+  <meta name="flags" content="dom">
+  <meta name="assert" content="Testing Serialization of Style Rules">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <style id="s-0">
+    @namespace svg "http://www.w3.org/2000/svg";
+  </style>
+  <style id="s-1">
+    @import url('main.css');
+  </style>
+  <style id="s-2">
+    h1 { background: indianred; }
+  </style>
+  <!-- Unsure why the second of 2 rules is the test subject in #s-3 -->
+  <style id="s-3">
+    @namespace svg "http://www.w3.org/2000/svg";
+    svg|a { color: white; }
+  </style>
+  <style id="s-4">
+    @font-face {
+      font-family: 'Megalopolis';
+      src: url('fonts/megalopolisextra-webfont.eot');
+      src: url('fonts/megalopolisextra-webfont.eot?#iefix') format('embedded-opentype'),
+           url('fonts/megalopolisextra-webfont.woff') format('woff'),
+           url('fonts/megalopolisextra-webfont.ttf') format('truetype'),
+           url('fonts/megalopolisextra-webfont.svg#MEgalopolisExtraRegular') format('svg');
+      font-weight: normal;
+      font-style: normal;
+    }
+  </style>
+  <style id="s-5">
+    @media (min-width: 200px) {
+        h1 { background: aliceblue; }
+    }
+  </style>
+  <style id="s-6">
+    @page :first {
+        h1 { color: #444; }
+    }
+  </style>
+</head>
+<body>
+  <div id="log"></div>
+  <script type="text/javascript">
+    "use strict";
+
+    var stylesheets = document.styleSheets;
+
+    var expectedTypes = [10, 3, 1, 1, 5, 4, 6];
+
+    var typesText = {
+      1: 'style rule',
+      3: '@import rule',
+      4: 'media rule',
+      5: '@font-face rule',
+      6: 'page rule',
+      10: 'namespace rule',
+      11: 'unrecognized rule'
+    };
+
+    for (var i = 0; i < stylesheets.length; i++) {
+      test( function () {
```

Comment:
I can see a couple paths forward here. The original test is looking at 2 different things: 1) that the stylesheets are applied in the correct order and 2) that the evaluated types are correct. However, to test ORDER, it's actually checking to see if the evaluated TYPES are in the expected order, given the rules in the stylesheets. I think there could be a better way to test order, but that would be out of the scope of this particular PR.

Knowing this, the 2 options forward in my mind:

1) Keep the tests separate, and in future find a better way to test order (maybe check to see that `.cssText` returns the expected rule?)
2) Recombine and not worry too much about update the rule order functionality; other fish to fry etc.

@gsnedders thoughts?